must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -258,6 +258,13 @@ for ns in $namespaces; do
             done
         done
 
+        # CRI-O have a limitation to upper limit to number of PIDs, so we found that when `ps aux | wc -l` exceeds 115 the resource cannot be collected
+        # hence to keep a buffer, we are waiting for 2 seconds until we have PIDs available, https://access.redhat.com/solutions/5597061
+        while [ "$(ps aux | wc -l)" -gt 100 ]
+            do
+               printf "waiting for PIDs to be empty before proceeding \n" | tee -a  "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
+               sleep 2
+        done
```

Comment:
Does it have to be at this specific point?
When do we hit this limit? Are there other alternatives?

Reply:
yes, before this point everything gets collected and there is no error but also the mirroring details are not getting collected.
With this, the details get collected as expected.